docs/source/conf.py

```diff
@@ -31,7 +31,7 @@
 release = "9.4.0-dev"
 
 # PROJ-data version
-data_version = "1.15"
+data_version = "1.16"
```

Comment:
also needs to be adujusted in data/sql/metadata.sql

Reply:
done